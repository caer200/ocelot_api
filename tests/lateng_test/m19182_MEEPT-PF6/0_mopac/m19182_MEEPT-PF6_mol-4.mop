CHARGES

 
H	2.608277 +1 1.407978 +1 10.020866 +1
C	2.990157 +1 1.383424 +1 10.890273 +1
C	4.379006 +1 1.585398 +1 11.047044 +1
C	2.182604 +1 1.152403 +1 11.979991 +1
C	4.885367 +1 1.586074 +1 12.368308 +1
N	5.208097 +1 1.763730 +1 9.936984 +1
H	1.254551 +1 0.999220 +1 11.846782 +1
C	2.699622 +1 1.137542 +1 13.280762 +1
C	4.041918 +1 1.361808 +1 13.472633 +1
S	6.548485 +1 1.768459 +1 12.773513 +1
C	6.574510 +1 2.011639 +1 10.008217 +1
C	4.610312 +1 1.466510 +1 8.597441 +1
H	2.128973 +1 0.974882 +1 14.022563 +1
H	4.401037 +1 1.365875 +1 14.352214 +1
C	7.283797 +1 2.030553 +1 11.235043 +1
C	7.323508 +1 2.250091 +1 8.828605 +1
H	3.930493 +1 0.755000 +1 8.705645 +1
H	5.320968 +1 1.113440 +1 8.005575 +1
C	3.962497 +1 2.654715 +1 7.919131 +1
C	8.673767 +1 2.223070 +1 11.263251 +1
H	6.879258 +1 2.275023 +1 7.989241 +1
C	8.682854 +1 2.444634 +1 8.880992 +1
H	3.442226 +1 2.351280 +1 7.133414 +1
H	3.343410 +1 3.104584 +1 8.547090 +1
O	4.974996 +1 3.559885 +1 7.507579 +1
H	9.134630 +1 2.214485 +1 12.093844 +1
C	9.370715 +1 2.424370 +1 10.099811 +1
H	9.164088 +1 2.595332 +1 8.075914 +1
C	4.460782 +1 4.607585 +1 6.673603 +1
H	10.312308 +1 2.548364 +1 10.120597 +1
H	3.994849 +1 5.280958 +1 7.229926 +1
H	3.809299 +1 4.233764 +1 6.028632 +1
C	5.595440 +1 5.256066 +1 5.933081 +1
H	6.131180 +1 4.564117 +1 5.470075 +1
H	5.237420 +1 5.877890 +1 5.251100 +1
O	6.418810 +1 5.968718 +1 6.847933 +1
C	7.510729 +1 6.572615 +1 6.177140 +1
H	8.024202 +1 5.884666 +1 5.704081 +1
H	8.088389 +1 7.019762 +1 6.830367 +1
H	7.175144 +1 7.230390 +1 5.533187 +1
