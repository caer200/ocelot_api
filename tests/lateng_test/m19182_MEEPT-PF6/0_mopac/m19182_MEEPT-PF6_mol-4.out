 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 21:31:49 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          2.60827700  *   1.40797800  *  10.02086600  *
     2       C          2.99015700  *   1.38342400  *  10.89027300  *
     3       C          4.37900600  *   1.58539800  *  11.04704400  *
     4       C          2.18260400  *   1.15240300  *  11.97999100  *
     5       C          4.88536700  *   1.58607400  *  12.36830800  *
     6       N          5.20809700  *   1.76373000  *   9.93698400  *
     7       H          1.25455100  *   0.99922000  *  11.84678200  *
     8       C          2.69962200  *   1.13754200  *  13.28076200  *
     9       C          4.04191800  *   1.36180800  *  13.47263300  *
    10       S          6.54848500  *   1.76845900  *  12.77351300  *
    11       C          6.57451000  *   2.01163900  *  10.00821700  *
    12       C          4.61031200  *   1.46651000  *   8.59744100  *
    13       H          2.12897300  *   0.97488200  *  14.02256300  *
    14       H          4.40103700  *   1.36587500  *  14.35221400  *
    15       C          7.28379700  *   2.03055300  *  11.23504300  *
    16       C          7.32350800  *   2.25009100  *   8.82860500  *
    17       H          3.93049300  *   0.75500000  *   8.70564500  *
    18       H          5.32096800  *   1.11344000  *   8.00557500  *
    19       C          3.96249700  *   2.65471500  *   7.91913100  *
    20       C          8.67376700  *   2.22307000  *  11.26325100  *
    21       H          6.87925800  *   2.27502300  *   7.98924100  *
    22       C          8.68285400  *   2.44463400  *   8.88099200  *
    23       H          3.44222600  *   2.35128000  *   7.13341400  *
    24       H          3.34341000  *   3.10458400  *   8.54709000  *
    25       O          4.97499600  *   3.55988500  *   7.50757900  *
    26       H          9.13463000  *   2.21448500  *  12.09384400  *
    27       C          9.37071500  *   2.42437000  *  10.09981100  *
    28       H          9.16408800  *   2.59533200  *   8.07591400  *
    29       C          4.46078200  *   4.60758500  *   6.67360300  *
    30       H         10.31230800  *   2.54836400  *  10.12059700  *
    31       H          3.99484900  *   5.28095800  *   7.22992600  *
    32       H          3.80929900  *   4.23376400  *   6.02863200  *
    33       C          5.59544000  *   5.25606600  *   5.93308100  *
    34       H          6.13118000  *   4.56411700  *   5.47007500  *
    35       H          5.23742000  *   5.87789000  *   5.25110000  *
    36       O          6.41881000  *   5.96871800  *   6.84793300  *
    37       C          7.51072900  *   6.57261500  *   6.17714000  *
    38       H          8.02420200  *   5.88466600  *   5.70408100  *
    39       H          8.08838900  *   7.01976200  *   6.83036700  *
    40       H          7.17514400  *   7.23039000  *   5.53318700  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          2.6083      1.4080     10.0209
     2         C          2.9902      1.3834     10.8903
     3         C          4.3790      1.5854     11.0470
     4         C          2.1826      1.1524     11.9800
     5         C          4.8854      1.5861     12.3683
     6         N          5.2081      1.7637      9.9370
     7         H          1.2546      0.9992     11.8468
     8         C          2.6996      1.1375     13.2808
     9         C          4.0419      1.3618     13.4726
    10         S          6.5485      1.7685     12.7735
    11         C          6.5745      2.0116     10.0082
    12         C          4.6103      1.4665      8.5974
    13         H          2.1290      0.9749     14.0226
    14         H          4.4010      1.3659     14.3522
    15         C          7.2838      2.0306     11.2350
    16         C          7.3235      2.2501      8.8286
    17         H          3.9305      0.7550      8.7056
    18         H          5.3210      1.1134      8.0056
    19         C          3.9625      2.6547      7.9191
    20         C          8.6738      2.2231     11.2633
    21         H          6.8793      2.2750      7.9892
    22         C          8.6829      2.4446      8.8810
    23         H          3.4422      2.3513      7.1334
    24         H          3.3434      3.1046      8.5471
    25         O          4.9750      3.5599      7.5076
    26         H          9.1346      2.2145     12.0938
    27         C          9.3707      2.4244     10.0998
    28         H          9.1641      2.5953      8.0759
    29         C          4.4608      4.6076      6.6736
    30         H         10.3123      2.5484     10.1206
    31         H          3.9948      5.2810      7.2299
    32         H          3.8093      4.2338      6.0286
    33         C          5.5954      5.2561      5.9331
    34         H          6.1312      4.5641      5.4701
    35         H          5.2374      5.8779      5.2511
    36         O          6.4188      5.9687      6.8479
    37         C          7.5107      6.5726      6.1771
    38         H          8.0242      5.8847      5.7041
    39         H          8.0884      7.0198      6.8304
    40         H          7.1751      7.2304      5.5332

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C17 H19 N O2 S  =    40 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
